vault-github/run.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
This is probably my bad for suggesting the use of `pushd`/`popd` previously, forgot it's a Bashism. I think it's better to keep all modules limited to `sh` functionality, that way we have the best compatibility, sometimes Bash may not be present.

The `pushd`/`popd` commands can be replaced by `cd "..."`/`cd -`, or a subshell form like this:

```sh
if ! (
	cd /tmp/...
	install
); then
	echo install fail
fi
# no need to cd here because cd was contained in subshell ( ... ).
```